lib/WP_Auth0_LoginManager.php

```diff
@@ -560,12 +560,12 @@ private function do_login( $user, $userinfo, $is_new, $id_token, $access_token,
 	 * @link https://codex.wordpress.org/Plugin_API/Action_Reference/wp_logout
 	 */
 	public function logout() {
-		$is_sso        = (bool) $this->a0_options->get( 'sso' );
-		$is_slo        = (bool) $this->a0_options->get( 'singlelogout' );
-		$is_auto_login = (bool) $this->a0_options->get( 'auto_login' );
+		if ( ! WP_Auth0::ready() ) {
+			return;
+		}
 
 		// If SSO/SLO is in use, redirect to Auth0 to logout there as well.
-		if ( $is_sso || $is_slo ) {
+		if ( $this->a0_options->get( 'sso' ) || $this->a0_options->get( 'singlelogout' ) ) {
```

Comment:
the auto-login check is casting to `bool`, why these aren't?

Reply:
None of them need to, will fix auto-login